### PR TITLE
Update (r)igm docs to reflect provider differences updating

### PR DESCRIPTION
**Pull request by rileykarson**
Make it clear what's available at GA and what isn't

-----------------------------------------------------------------
# [all]
## [terraform]
Update igm and rigm docs

### CODE REVIEW

File: provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 1]
```diff
@@ -150,7 +151,7 @@ This property is in beta, and should be used with the terraform-provider-google-
 See [Provider Versions](https://terraform.io/docs/providers/google/provider_versions.html) for more details on beta fields.
 - - -
 
-The **rolling_update_policy** block supports:
+The `update_policy` block supports:
 
 ```hcl
 rolling_update_policy{
```
Comment (danawillow) [inline provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: can you update this example too (and in the other file)?

File: provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 2]
```diff
@@ -150,7 +151,7 @@ This property is in beta, and should be used with the terraform-provider-google-
 See [Provider Versions](https://terraform.io/docs/providers/google/provider_versions.html) for more details on beta fields.
```
Comment (danawillow) [inline provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: Line 148: rm rolling_

File: provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 3]
```diff
@@ -90,14 +90,15 @@ The following arguments are supported:
...
+    which all new instances will be created. This field is only present in the
+    `google` provider.
 
 * `version` - (Optional) Application versions managed by this instance group. Each
     version deals with a specific instance template, allowing canary release scenarios.
-    Conflicts with `instance_template`. Structure is documented below. Beware that
-    exactly one version must not specify a target size. It means that versions with
-    a target size will respect the setting, and the one without target size will
-    be applied to all remaining Instances (top level target_size - each version target_size).
+    Structure is documented below. Beware that exactly one version must not specify a
```
Comment (danawillow) [inline provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: While you're here, any thoughts on a better way to word this? It's fairly hard to understand as-is (no worries if not, I know you didn't write it)

Comment (rileykarson) [reply provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: Sure, I took a crack at it - let me know your thoughts.

Comment (danawillow) [reply provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: That's a lot better, thanks!

**Pull request merged**